src/printer-postcss.js

```diff
@@ -69,13 +69,19 @@ function genericPrint(path, options, print) {
         n.value.group.type === "value-value" &&
         n.value.group.group.type === "value-func" &&
         n.value.group.group.value === "extend";
+      const isComposed =
+        n.value.type === "value-root" &&
+        n.value.group.type === "value-value" &&
+        n.prop === "composes";
 
       return concat([
         n.raws.before.replace(/[\s;]/g, ""),
         n.prop,
         ":",
         isValueExtend ? "" : " ",
-        path.call(print, "value"),
+        isComposed
+          ? n.value.group.group.groups.join("")
+          : path.call(print, "value"),
```

Comment:
Sorry, hopefully used it correctly now and it seems to do the trick. Got a bit "confused" by the line location of the comment as I would expect that we'd want to keep "normal" formatting without removing lines in case it's not a 'composes'?